code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml

```diff
@@ -126,7 +126,8 @@
 
     <!-- Load Balancer Target -->
     <process:process name="loadbalancertarget.create" resourceType="loadBalancerTarget" start="requested" transitioning="activating" done="active" />
-    <process:process name="loadbalancertarget.remove" resourceType="loadBalancerTarget" start="active,activating" transitioning="removing" done="removed" />
+    <process:process name="loadbalancertarget.remove" resourceType="loadBalancerTarget" start="active,activating,updating-active" transitioning="removing" done="removed" />
+    <process:process name="loadbalancertarget.update" resourceType="loadBalancerTarget" start="active,activating" transitioning="updating-active" done="active" />
```

Comment:
@alena1108 are you sure you want to be able to call update while the resource is in the "activating" state?
That is not standard. The standard way only allows updating from the resting states of active and inactive.